plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -62,7 +62,7 @@ func (r *RBACAuthorizer) Authorize(attr authorizer.Attributes) error {
 		}
 	}
 
-	return validation.ConfirmNoEscalation(ctx, r.authorizationRuleResolver, []rbac.PolicyRule{requestedRule})
+	return validation.ConfirmNoEscalation(ctx, r.authorizationRuleResolver, []rbac.PolicyRule{requestedRule}), nil
```

Comment:
@ericchiang synthesizing a rule from attributes and doing an escalation check on that rule seems both expensive (since the escalation check is geared toward many-to-many permission checks) and error prone (looking at this more, the non-resource-url rule is not built correctly since it leaves out verb). to avoid hijacking this PR, we can split this discussion into a separate issue